astropy/modeling/tests/test_models.py

```diff
@@ -421,12 +428,17 @@ def test_fitter1D(self, model_class, test_parameters):
 
     @pytest.mark.skipif('not HAS_SCIPY')
     @pytest.mark.filterwarnings(r'ignore:.*:RuntimeWarning')
-    def test_deriv_1D(self, model_class, test_parameters):
+    @pytest.mark.parametrize('fitter', fitters)
+    def test_deriv_1D(self, model_class, test_parameters, fitter):
         """
         Test the derivative of a model by comparing results with an estimated
         derivative.
         """
 
+        if fitter == fitting.TRFLSQFitter and \
+                model_class in (models.ExponentialCutoffPowerLaw1D, models.PowerLaw1D):
+            pytest.xfail("TRFLSQFitter seems to be broken for this test.")
+
```

Comment:
@krachyon these two cases with the `TRFLSQFitter` fail as well. I have not investigated why they fail yet.